### PR TITLE
enable publishing of alerting-spi jar

### PR DESCRIPTION
**Pull request by sbcd90**
*Issue #, if available:*
#1546 

*Description of changes:*
enable publishing of alerting-spi jar

*CheckList:*
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/alerting/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by eirsep — APPROVED** [review]

File: spi/build.gradle [hunk 1]
```diff
@@ -3,14 +3,90 @@
...
+                        organization = "Amazon Web Services"
+                        organizationUrl = "https://aws.amazon.com"
+                    }
+                }
+            }
+        }
+    }
+
+    gradle.startParameter.setShowStacktrace(ShowStacktrace.ALWAYS)
+    gradle.startParameter.setLogLevel(LogLevel.DEBUG)
```
Comment (lezzago) [inline spi/build.gradle:91]: Will this cause the logs to always be at debug level?

Comment (sbcd90) [reply spi/build.gradle:91]: this will just set the log level for gradle tasks.

File: spi/build.gradle [hunk 2]
```diff
@@ -3,14 +3,90 @@
...
+                    license {
+                        name = "The Apache License, Version 2.0"
+                        url = "http://www.apache.org/licenses/LICENSE-2.0.txt"
+                    }
+                }
+                developers {
+                    developer {
+                        id = "amazonwebservices"
+                        organization = "Amazon Web Services"
+                        organizationUrl = "https://aws.amazon.com"
```
Comment (lezzago) [inline spi/build.gradle:83]: Shouldn't this be opensearch and linking to opensearch url?

Comment (sbcd90) [reply spi/build.gradle]: fixed it.

**Review by lezzago — APPROVED** [review]

**Pull request merged**